### DISCUSSION TITLE
Bereinigung der Branches

### DISCUSSION THREAD

**Discussion by Philip-Wiege** [open3e/open3e#146] [Ideas]
Hallo zusammen,

ich würde einfach mal eine Bereinigung der Branches in den Raum stellen. Macht es Sinn, die Branches experimental und no_pip zu behalten? Ist dort noch etwas stabiles drin, was wir in develop oder master mergen müssten?

Grüße
Philip

**Comment by MyHomeMyData** [comment]
@Philip-Wiege Sehr guter Vorschlag. "experimental" hatte ich mal angelegt. Ist obsolet, habe ich eben gelöscht. "service77" ist komplett in "develop" eingeflossen, habe ich auch gelöscht.

Bei "no_pip" bin ich etwas hin und her gerissen. Brauchen wir zwar nicht, aber es ist der letzte Stand vor dem Umstieg auf Installation per pip. Falls man damit nochmal was mit der alten Installationsmethode ausprobieren möchte, könnte der Stand helfen.

@philippoo66 Wie siehst Du das?

**Comment by Philip-Wiege** [comment]
Das mit no_pip leuchtet ein. Kann man daraus nicht einen Tag / Release machen? Damit werden doch bestimmte wichtige Punkte im Verlauf des Projektes gekennzeichnet?

**Reply by MyHomeMyData** [reply]
Ja, vermutlich geht das. Da kenne mich aber mit Git zu wenig aus. Kann auch die Vor- und Nachteile nicht einschätzen.

Kennst Du Dich da aus?

**Reply by philippoo66** [reply]
das no_pip würde ich erstmal noch behalten. das frisst ja kein Gras.
